cpp/modules/deck.gl/core/src/lib/attribute/attribute-manager.cc

```diff
@@ -46,20 +47,22 @@ void AttributeManager::invalidateAll() {
   probegl::DebugLog() << "AttributeManager: invalidating all attributes";
 }
 
-auto AttributeManager::update(const std::shared_ptr<arrow::Table>& table) -> std::shared_ptr<arrow::Table> {
+auto AttributeManager::update(const std::shared_ptr<arrow::Table>& table) -> std::shared_ptr<WebGPUTable> {
   // Create an empty output table
   std::vector<std::shared_ptr<arrow::Field>> fields{};
-  std::vector<std::shared_ptr<arrow::Array>> arrays{};
+  std::vector<std::shared_ptr<WebGPUColumn>> columns{};
 
   // Iterate over descriptors and create one column per-descriptor
   for (auto descriptor : this->_descriptors) {
     auto field = std::make_shared<arrow::Field>(descriptor->name, descriptor->type);
     fields.push_back(field);
```

Comment:
What happens to the fields? As you know I think we should still create a schema and store on the GPUTable...